hortonworks-on-centos/azuredeploy.json

```diff
@@ -126,7 +126,7 @@
             "nameNodeCount": 1,
             "dataNodeCount": 3,
             "masterVmSize": "Standard_A3",            
-            "masterNodeTemplateUri": "/cluster-node-16disk.json",
+            "masterNodeTemplateUri": "/cluster-node-8disk.json",
             "workerVmSize": "Standard_A3",
```

Comment:
Not sure what you mean by D, but this change is correct.  It should have been using only 8 disks with the A3 VM